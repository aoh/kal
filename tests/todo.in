# flags: -t 1000000 -n 1
1.1.1970
 + let there be time

13.1.1970
 + does not happen yet

every day: foo
