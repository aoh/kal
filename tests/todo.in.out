12.1.1970, Monday, week 3
 - foo
 + let there be time
